charts/producer-app-cleanup-job/templates/job.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     {{- end }}
 spec:
   ttlSecondsAfterFinished: 30
-  replicas: {{ .Values.replicaCount }}
+  parallelism: {{ .Values.replicaCount }}
```

Comment:
Replicas for our jobs is actually wrong, just remove the variable (also check the streams job)

Reply:
Helm 3 is validating the openapi now before deploying so that something like this results in an error